dmoj/commands/test.py

```diff
@@ -0,0 +1,87 @@
+import sys
+import traceback
+from operator import itemgetter
+
+from dmoj import executors
+from dmoj.commands.base_command import Command
+from dmoj.error import InvalidCommandException
+from dmoj.judgeenv import get_problem_root, get_supported_problems
+from dmoj.problem import Problem
+from dmoj.testsuite import Tester
+from dmoj.utils.ansi import ansi_style, print_ansi
+
+all_executors = executors.executors
+
+
+class ProblemTester(Tester):
+    def test_problem(self, problem_id):
+        self.output(ansi_style('Testing problem #ansi[%s](cyan|bold)...') % problem_id)
+        fails = 0
+
+        config = Problem.get_config(problem_id)
+
+        if not config or 'tests' not in config or not config['tests']:
+            self.output(ansi_style('\t#ansi[Skipped](magenta|bold) - No tests found'))
+
+        for test in config['tests']:
+            # Do this check here as we need some way to identify the test
+            if 'source' not in test:
+                continue
+
+            test_name = test.get('label', test['source'])
+            self.output(ansi_style('\tRunning test #ansi[%s](yellow|bold)') % test_name)
+            try:
+                test_fails = self.run_test(problem_id, test)
```

Comment:
`fails +=` here seems like it'd be equivalent, and not require the `test_fails` var?

Reply:
`test_fails` is required 7 lines later

Reply:
Ahh, I see the `not test_fails` now.